xml/System.Security.Cryptography/DSAOpenSsl.xml

```diff
@@ -126,9 +182,23 @@
         <Parameter Name="pkeyHandle" Type="System.Security.Cryptography.SafeEvpPKeyHandle" />
       </Parameters>
       <Docs>
-        <param name="pkeyHandle">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="pkeyHandle">The OpenSSL <c>EVP_PKEY*</c> value to use as the key, represented as a <see cref="T:System.Security.Cryptography.SafeEvpPKeyHandle" />.</param>
```

Comment:
Same question as above applies for `EVP_PKEY*`. Since I'm not too familiar with all these cryptography methods, and they are advanced, I am not sure if we should expect users using these APIs to already be familiar to what the wildcard is referring to.

Reply:
Not a wildcard, a pointer type.  These methods are pointer-based interop with the underlying system.